airflow/cli/cli_parser.py

```diff
@@ -778,6 +782,12 @@ class GroupCommand(NamedTuple):
         func=lazy_load_command('airflow.cli.commands.dag_command.dag_list_dags'),
         args=(ARG_SUBDIR, ARG_OUTPUT),
     ),
+    ActionCommand(
+        name='generate_kubernetes_pod_yaml',
+        help="Generate YAML files for all tasks in DAG",
+        func=lazy_load_command('airflow.cli.commands.dag_command.generate_pod_yaml'),
```

Comment:
```suggestion
        name='dump-pod-yaml',
        help="Generate YAML files with K8S Pod definitions.",
        description="Generate YAML files with Kubernetes Pod definitions for all tasks in DAG",
```
This command name is too long. It doesn't look good in CLI.
<img width="687" alt="Screenshot 2020-09-04 at 00 25 43" src="https://user-images.githubusercontent.com/12058428/92179595-440bce80-ee45-11ea-8f0b-ed578042e3c3.png">